src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseBase.cpp

```diff
@@ -157,17 +235,36 @@ void SinglePhaseBase::updateFluidModel( ObjectManagerBase & dataGroup ) const
   GEOSX_MARK_FUNCTION;
 
   arrayView1d< real64 const > const pres = dataGroup.getExtrinsicData< extrinsicMeshData::flow::pressure >();
+  arrayView1d< real64 const > const temp = dataGroup.getExtrinsicData< extrinsicMeshData::flow::temperature >();
 
   SingleFluidBase & fluid =
     getConstitutiveModel< SingleFluidBase >( dataGroup, dataGroup.getReference< string >( viewKeyStruct::fluidNamesString() ) );
 
   constitutiveUpdatePassThru( fluid, [&]( auto & castedFluid )
   {
     typename TYPEOFREF( castedFluid ) ::KernelWrapper fluidWrapper = castedFluid.createKernelWrapper();
-    FluidUpdateKernel::launch( fluidWrapper, pres );
+    thermalSinglePhaseBaseKernels::FluidUpdateKernel::launch( fluidWrapper, pres, temp );
```

Comment:
Are we now only ever calling the temperature-dependent fluid updates? Do we want to remove the other versions?

Reply:
Just confirmed that we are now only calling the fluid update that has the temperature input. For the isothermal fluid model, the temperature input is unused. I will then remove the original fluid update function without a temperature input. 

> > This looks good to me at a high level. I've been removed from physics solver development for so long that it's now difficult for me to review in detail :)
> > I wonder if we could eventually converge on a fluid design similar to `CO2BrineFluid`: parametrize `SinglePhaseFluid` on `PhaseModel` (can literally reuse the same type) which is in turn parametrized on density, viscosity and enthalpy models, and the latter can be either a `NoOpFunction` for isothermal or an `ExponentialRelation`... or maybe something else. I also think that viscosity dependence on temperature needs to eventually be modeled, though current exponential relations don't apply there. This are all future ideas, no need for changes right now.
> > Also note that his PR updates BLT which is also updated in #1949. If that was intentional, make sure there are no conflicts.
> 
> @frankfeifan is the BLT change intentional or is it simply the result of a forgotten `git submodule update`?

It is NOT intentional. I think I forgot 'git submodule update'.